src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithDefaultNullFW.java

```diff
@@ -0,0 +1,387 @@
+                fieldLimit = variantOfString1RO.limit();
+                bitmask |= 1 << INDEX_VARIANT_OF_STRING1;
+                break;
+            case INDEX_VARIANT_OF_STRING2:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    if (variantOfString2RO.tryWrap(buffer, fieldLimit, maxLimit) == null)
+                    {
+                        return null;
+                    }
+                    fieldLimit = variantOfString2RO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_STRING2;
+                }
+                else
+                {
+                    fieldLimit++;
+                }
+                break;
+            case INDEX_VARIANT_OF_UINT:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    if (variantOfUintRO.tryWrap(buffer, fieldLimit, maxLimit) == null)
+                    {
+                        return null;
+                    }
+                    fieldLimit = variantOfUintRO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_UINT;
+                }
+                else
+                {
+                    fieldLimit++;
+                }
+                break;
+            case INDEX_VARIANT_OF_INT:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    if (variantOfIntRO.tryWrap(buffer, fieldLimit, maxLimit) == null)
+                    {
+                        return null;
+                    }
+                    fieldLimit = variantOfIntRO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_INT;
+                }
+                else
+                {
+                    fieldLimit++;
+                }
+                break;
+            }
+        }
+        if (limit() > maxLimit)
+        {
+            return null;
+        }
+        return this;
+    }
+
+    @Override
+    public int limit()
+    {
+        return offset() + buffer().getInt(offset() + PHYSICAL_LENGTH_OFFSET);
+    }
+
+    @Override
+    public String toString()
+    {
+        boolean variantOfString2IsSet = (bitmask & MASK_VARIANT_OF_STRING2) != 0;
+        boolean variantOfIntIsSet = (bitmask & MASK_VARIANT_OF_INT) != 0;
+
+        StringBuilder format = new StringBuilder();
+        format.append("LIST_WITH_DEFAULT_NULL [");
+        format.append("variantOfString1={0}");
+        if (variantOfString2IsSet)
+        {
+            format.append(", variantOfString2={1}");
+        }
+        format.append(", variantOfUint={2}");
+        if (variantOfIntIsSet)
+        {
+            format.append(", variantOfInt={3}");
+        }
+        format.append("]");
+        return MessageFormat.format(format.toString(),
+            variantOfString1(),
+            variantOfString2IsSet ? variantOfString2() : null,
+            variantOfUint(),
+            variantOfIntIsSet ? variantOfInt() : null);
+    }
```

Comment:
Suggest a list of optional fields at the top of this method, defaulted to `null`, such as:
```
Object variantOfString2 = null;
Object variantOfInt = null;

StringBuilder format = new StringBuilder();
format.append("LIST_WITH_DEFAULT_NULL [");
format.append("variantOfString1={0}");
if ((bitmask & MASK_VARIANT_OF_STRING2) != 0L)
{
    format.append(", variantOfString2={1}");
    variantOfString2 = variantOfString2();
}
format.append(", variantOfUint={2}");
if ((bitmask & MASK_VARIANT_OF_INT) != 0L)
{
    format.append(", variantOfInt={3}");
    variantOfInt = variantOfInt();
}
format.append("]");
return MessageFormat.format(format.toString(),
            variantOfString1(),
            variantOfString2,
            variantOfUint(),
            variantOfInt);
```